### DISCUSSION TITLE
Single file exporter.

### DISCUSSION THREAD

**Discussion by scolastico** [nuxt/nuxt#17403] [General]
Because i realy need a single file export (inlined js, and css) i created a small module. Its not perfekt but does the job.

Related to: https://github.com/nuxt/framework/discussions/5497

Github: https://github.com/scolastico/nuxt-single-file-exporter
NPM: https://www.npmjs.com/package/nuxt-single-file-exporter

**Comment by Flanker72** [comment]
Great result, @scolastico! That's very handy module 👏👏👏

**Reply by scolastico** [reply]
Yeah but currently it cant be loaded via npm but i will fix this is a few days but in the meantime you can simply copy the src/module folder
